xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/export_modal.vm

```diff
@@ -4,70 +4,82 @@
-        #if("$!{services.officemanager.serverState}" == 'Connected')
-          <a href="$doc.getURL('export', "format=odt&amp;$!escapetool.html($request.getQueryString())")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.odt')</a>
-          <a href="$doc.getURL('export', "format=rtf&amp;$!escapetool.html($request.getQueryString())")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.rtf')</a>
-        #end
-        #foreach ($exportExtension in $exportExtensions)
-          #if ($exportExtension.parameters.category == 'office')
-            $services.rendering.render($exportExtension.execute(), 'html/5.0')
-          #end
+  <div class="panel-heading" role="tab" id="exportModalHeadingOffice">
+    <h4 class="panel-title">
+      <a role="button" data-toggle="collapse" data-parent="#exportModelAccordion" href="#exportModelOfficeCollapse"
+         aria-expanded="true" aria-controls="exportModelOfficeCollapse">
+        $escapetool.xml($services.localization.render('core.exporter.headings.officeFormats'))
+      </a>
+    </h4>
+  </div>
+  <div id="exportModelOfficeCollapse" class="panel-collapse collapse in" role="tabpanel"
+       aria-labelledby="exportModalHeadingOffice">
+    <div class="panel-body">
+      <a href="$doc.getURL('view', "xpage=pdfoptions&amp;qs=$!escapetool.url($request.getQueryString())")"
+         class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.pdf')</a>
+      #if("$!{services.officemanager.serverState}" == 'Connected')
+        <a href="$doc.getURL('export', "format=odt&amp;$!escapetool.html($request.getQueryString())")"
+           class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.odt')</a>
+        <a href="$doc.getURL('export', "format=rtf&amp;$!escapetool.html($request.getQueryString())")"
+           class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.rtf')</a>
+      #end
+      #foreach ($exportExtension in $exportExtensions)
+        #if ($exportExtension.parameters.category == 'office')
+          $services.rendering.render($exportExtension.execute(), 'html/5.0')
         #end
-      </div>
+      #end
     </div>
   </div>
+</div>
 #end
 ##
 #macro (displayOtherFormats $exportExtensions)
   <div class="panel panel-default">
-    <div class="panel-heading" role="tab" id="exportModalHeadingOther">
-      <h4 class="panel-title">
-        <a class="collapsed" role="button" data-toggle="collapse" data-parent="#exportModelAccordion" href="#exportModelOtherCollapse" aria-expanded="false" aria-controls="exportModelOtherCollapse">
-          $escapetool.xml($services.localization.render('core.exporter.headings.otherFormats'))
-        </a>
-      </h4>
-    </div>
-    <div id="exportModelOtherCollapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="exportModalHeadingOther">
-      <div class="panel-body">
```

Comment:
Same. I think the previous indentation was good.

Reply:
@mflorea here I just added the form around the div but apparently git has some trouble to process a good diff there...

Reply:
You could try

    <form id="export-form" method="post" class="panel-body">

i.e. convert the panel body DIV into a form instead of wrapping it. This way you don't need to change the indentation.

Reply:
Hm. Didn't thought about that. I'll try.

Reply:
Indeed it fixes the diff.